pulsar-client-cpp/python/CMakeLists.txt

```diff
@@ -0,0 +1,35 @@
+#
+# Copyright 2016 Yahoo Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+INCLUDE_DIRECTORIES("${Boost_INCLUDE_DIRS}" "${PYTHON_INCLUDE_DIRS}")
+
+ADD_LIBRARY(_pulsar SHARED src/pulsar.cc src/producer.cc src/consumer.cc
+                            src/config.cc src/enums.cc src/client.cc
+                            src/message.cc src/authentication.cc)
+SET(CMAKE_SHARED_LIBRARY_PREFIX )
+SET(CMAKE_SHARED_LIBRARY_SUFFIX .so)
+
+set (CMAKE_SHARED_LINKER_FLAGS " -static -undefined dynamic_lookup ")
```

Comment:
Yes, this is to avoid linking with Python libs (since the wrapper will be loaded from inside python interpreter anyway)